godot-macros/src/class/data_models/inherent_impl.rs

```diff
@@ -89,6 +93,19 @@ pub fn transform_inherent_impl(
     #[cfg(not(all(feature = "register-docs", since_api = "4.3")))]
     let docs = quote! {};
 
+    // This is the container struct that holds the names of all registered #[func]s.
+    // (The struct is declared by the macro derive_godot_class.)
+    let class_functions_name = format_func_name_struct_name(&class_name);
+    // As the impl block could be of the form "path::class", and we add a second impl block below, we need the full path, not just the class name.
+    let this_class_full_path = impl_block.self_ty.as_path().ok_or(error_fn(
+        "unexpected: the function already checked 'as_path' above in validate_impl",
+        &impl_block,
+    ))?;
+    let class_functions_path: Path =
+        replace_class_in_path(this_class_full_path, class_functions_name);
+    // For each #[func] in this impl block, we create one constant.
```

Comment:
Kinda noisy; personally I would rather see proper docstring for `make_func_name_constants` . 

Also why is it named make_**func**_name_constants when it expects **funcs** as an argument :thinking:?

Reply:
> Also why is it named make_func_name_constants when it expects funcs as an argument 🤔?

One constant is a ``func_name_constant``, multiple constants are ``func_name_constant[s]``, at least that was my logic.